docs/integrations/persisting-store-data.md

```diff
@@ -731,3 +731,32 @@ interface BearState {
     removeItem: (name) => localStorage.removeItem(name),
   },
 ```
+
+If writing serialisation and deserialisation code is tedious, you can use third-party libraries to serialise and deserialise different types of data.
```

Comment:
Please use the American English spelling — _serialization_ for this and other occurrences.